bigtable/client/internal/table_admin.h

```diff
@@ -113,6 +117,12 @@ class TableAdmin {
     return instance_name() + "/tables/" + table_id;
   }
 
+  /// Return the fully qualified name of a snapshot.
+  std::string SnapshotName(std::string const& name,
```

Comment:
The order is backwards, which I find confusing, consider using `cluster_id` and `name` in that order here and everywhere.
Also consider using `snapshot_id` instead of `name` here.